File: src/main/java/hits/internship/NotificationService/model/kafka/ChangingPractise.java
package hits.internship.NotificationService.model.kafka;

import hits.internship.NotificationService.model.enumeration.StatusType;
import lombok.Data;
import lombok.EqualsAndHashCode;

@EqualsAndHashCode(callSuper = true)
@Data
public class ChangingPractise extends KafkaMessageRequest{
    String oldCompanyName;
    String oldPosition;
    String newCompanyName;
    String newPosition;
    StatusType newStatus;
}

File: dotnet/NotificationModule.Kafka/Messages/ChangingPractise.cs
using NotificationModule.Domain.Enums;

namespace NotificationModule.Kafka.Messages;

public class ChangingPractise : KafkaMessageRequest
{
    public string OldCompanyName { get; set; }
    public string NewCompanyName { get; set; }
    public string OldPosition { get; set; }
    public string NewPosition { get; set; }
    public ChangingPracticeStatusType NewStatus { get; set; }
}